storybook/stories/API/cartesian/Bar.stories.tsx

```diff
@@ -1,30 +1,438 @@
+  onAnimationIteration,
+  onAnimationIterationCapture,
+  onAnimationStart,
+  onAnimationStartCapture,
+  onAuxClick,
+  onAuxClickCapture,
+  onBeforeInput,
+  onBeforeInputCapture,
+  onBlur,
+  onBlurCapture,
+  onCanPlay,
+  onCanPlayCapture,
+  onCanPlayThrough,
+  onCanPlayThroughCapture,
+  onChange,
+  onChangeCapture,
+  onClick,
+  onClickCapture,
+  onCompositionEnd,
+  onCompositionEndCapture,
+  onCompositionStart,
+  onCompositionStartCapture,
+  onCompositionUpdate,
+  onCompositionUpdateCapture,
+  onContextMenu,
+  onContextMenuCapture,
+  onCopy,
+  onCopyCapture,
+  onCut,
+  onCutCapture,
+  onDoubleClick,
+  onDoubleClickCapture,
+  onDrag,
+  onDragCapture,
+  onDragEnd,
+  onDragEndCapture,
+  onDragEnter,
+  onDragEnterCapture,
+  onDragExit,
+  onDragExitCapture,
+  onDragLeave,
+  onDragLeaveCapture,
+  onDragOver,
+  onDragOverCapture,
+  onDragStart,
+  onDragStartCapture,
+  onDrop,
+  onDropCapture,
+  onDurationChange,
+  onDurationChangeCapture,
+  onEmptied,
+  onEmptiedCapture,
+  onEncrypted,
+  onEncryptedCapture,
+  onEnded,
+  onEndedCapture,
+  onError,
+  onErrorCapture,
+  onFocus,
+  onFocusCapture,
+  onGotPointerCapture,
+  onGotPointerCaptureCapture,
+  onInput,
+  onInputCapture,
+  onInvalid,
+  onInvalidCapture,
+  onKeyDown,
+  onKeyDownCapture,
+  onKeyPress,
+  onKeyPressCapture,
+  onKeyUp,
+  onKeyUpCapture,
+  onLoad,
+  onLoadCapture,
+  onLoadedData,
+  onLoadedDataCapture,
+  onLoadedMetadata,
+  onLoadedMetadataCapture,
+  onLoadStart,
+  onLoadStartCapture,
+  onLostPointerCapture,
+  onLostPointerCaptureCapture,
+  onMouseDown,
+  onMouseDownCapture,
+  onMouseEnter,
+  onMouseLeave,
+  onMouseMove,
+  onMouseMoveCapture,
+  onMouseOut,
+  onMouseOutCapture,
+  onMouseOver,
+  onMouseOverCapture,
+  onMouseUp,
+  onMouseUpCapture,
+  onPaste,
+  onPasteCapture,
+  onPause,
+  onPauseCapture,
+  onPlay,
+  onPlayCapture,
+  onPlaying,
+  onPlayingCapture,
+  onPointerCancel,
+  onPointerCancelCapture,
+  onPointerDown,
+  onPointerDownCapture,
+  onPointerEnter,
+  onPointerEnterCapture,
+  onPointerLeave,
+  onPointerLeaveCapture,
+  onPointerMove,
+  onPointerMoveCapture,
+  onPointerOut,
+  onPointerOutCapture,
+  onPointerOver,
+  onPointerOverCapture,
+  onPointerUp,
+  onPointerUpCapture,
+  onProgress,
+  onProgressCapture,
+  onRateChange,
+  onRateChangeCapture,
+  onReset,
+  onResetCapture,
+  onScroll,
+  onScrollCapture,
+  onSeeked,
+  onSeekedCapture,
+  onSeeking,
+  onSeekingCapture,
+  onSelect,
+  onSelectCapture,
+  onStalled,
+  onStalledCapture,
+  onSubmit,
+  onSubmitCapture,
+  onSuspend,
+  onSuspendCapture,
+  onTimeUpdate,
+  onTimeUpdateCapture,
+  onTouchCancel,
+  onTouchCancelCapture,
+  onTouchEnd,
+  onTouchEndCapture,
+  onTouchMove,
+  onTouchMoveCapture,
+  onTouchStart,
+  onTouchStartCapture,
+  onTransitionEnd,
+  onTransitionEndCapture,
+  onVolumeChange,
+  onVolumeChangeCapture,
+  onWaiting,
+  onWaitingCapture,
+  onWheel,
+  onWheelCapture,
```

Comment:
I so turns out that these are exactly no all event handlers. 
Going further I discourage importing groups of props from shared descriptions, because chances are that a component does not actually have the prop. 

In this case, some events showed up that were not supported. I did not identify which exactly, but merely compared total numbers, which were not identical.

Reply:
there has to be a cleaner way of doing this :o

Reply:
I was hoping for you to share one with me 😓